yarn.lock

```diff
@@ -2794,6 +2867,24 @@
   dependencies:
     svg2vectordrawable "^2.9.1"
 
+"@vector-im/compound-design-tokens@git+https://github.com/vector-im/compound-design-tokens.git#develop":
+  version "0.0.3"
+  resolved "git+https://github.com/vector-im/compound-design-tokens.git#f2199a9073a22989d1f04c6b4b344b9acc7f8673"
+  dependencies:
+    svg2vectordrawable "^2.9.1"
```

Comment:
Git dependencies are problematic in yarn due to them not caching consistently with other dependencies and being a major issue for some community packages https://github.com/yarnpkg/yarn/issues/4722 https://github.com/vector-im/element-web/issues/20628